exercises/04-integration/after/Divergent.Customers/Config/EndpointConfig.cs

```diff
@@ -48,8 +32,12 @@ public void Customize(EndpointConfiguration endpointConfiguration)
             endpointConfiguration.AuditProcessedMessagesTo("audit");
 
             var conventions = endpointConfiguration.Conventions();
-            conventions.DefiningCommandsAs(t => t.Namespace != null && t.Namespace.StartsWith("Divergent") && t.Namespace.EndsWith("Commands") && t.Name.EndsWith("Command"));
-            conventions.DefiningEventsAs(t => t.Namespace != null && t.Namespace.StartsWith("Divergent") && t.Namespace.EndsWith("Events") && t.Name.EndsWith("Event"));
+            conventions.DefiningCommandsAs(t =>
+                t.Namespace != null && t.Namespace.StartsWith("Divergent") && t.Namespace.EndsWith("Commands") &&
+                t.Name.EndsWith("Command"));
+            conventions.DefiningEventsAs(t =>
+                t.Namespace != null && t.Namespace.StartsWith("Divergent") && t.Namespace.EndsWith("Events") &&
+                t.Name.EndsWith("Event"));
```

Comment:
It took me a while to figure out that there are no actual changes here and it's just the addition of line breaks. This is a good change, but please make it in a separate refactoring commit if it is going to be in the new PR.